src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -397,6 +397,21 @@ internal Symbol BindNamespaceOrTypeOrAliasSymbol(ExpressionSyntax syntax, Diagno
 
         private TypeSymbol BindTupleType(TupleTypeSyntax syntax, DiagnosticBag diagnostics)
         {
+            // If the tuple type is bound in a declaration context then we must have the
```

Comment:
Technically we shouldn't need the attribute if the tuple involves no element names. 
In practice, this is not very important, since the attribute is provided in the same library as VT.
We may want to confirm with LDM, in case they have a stronger reaction.